### PR TITLE
<FIX> fixed launch url malfunction issue with IOS

### PR DESCRIPTION
**Pull request by samcho0608**
I was unable to open App Store URL with LaunchMode.externalNonBrowserApplication. Since the code update here with LaunchMode was to aid Android OS, I changed the code so that only Android will launch the URL with LaunchMode.externalNonBrowserApplication while others kept their LaunchMode.platformDefault.

### CODE REVIEW

**Comment by larryaasen** [comment]
This looks good. But there is one small change requested.

File: lib/src/upgrader.dart [hunk 1]
```diff
@@ -794,7 +795,9 @@ class Upgrader {
     if (await canLaunchUrl(Uri.parse(_appStoreListingURL!))) {
       try {
         await launchUrl(Uri.parse(_appStoreListingURL!),
-            mode: LaunchMode.externalNonBrowserApplication);
+            mode: Platform.isAndroid
```
Comment (larryaasen) [inline lib/src/upgrader.dart:798]:
```suggestion
            mode: UpgradeIO.isAndroid
```

File: lib/src/upgrader.dart [hunk 2]
```diff
@@ -3,6 +3,7 @@
  */
 
 import 'dart:async';
+import 'dart:io';
```
Comment (larryaasen) [inline lib/src/upgrader.dart:6]:
```suggestion
```

**Comment by larryaasen** [comment]
This LGTM. Thanks for the PR. Merging now.

**Comment by samcho0608** [comment]
You're welcome! Glad I can contribute

**Pull request merged**